### ISSUE TITLE
Feature Request: changelog

### ISSUE THREAD

**Issue by teetotum** [steve-py96/ahocon#4]
I'm maintaining a project that uses ahocon and I try to keep all dependencies updated. Currently when there is a new version of ahocon I have no clue as to what has changed w.r.t. the previous version and what to expect when upgrading.

A changelog would be a useful addition to the project.

I can recommend a rigid structure as described by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/); and I use it for [one](https://github.com/teetotum/axios-response-mock/blob/master/CHANGELOG.md) of my own OSS projects.

**Comment by steve-py96** [comment]
good point, I'll add one the upcoming days (sorry for the delay, my inbox was flooded with spam mails which i had to take care of before seeing my normal mails again 👀)
